app.py

```diff
@@ -150,19 +150,22 @@ def submit_blast():
 
 @app.route('/blast-vip')
 def the_blastvip_form():
-    form = BlastVIPForm()
-    if request.args.get('amount'):
-        amount = request.args.get('amount')
-    else:
-        amount = 275
-    installment_period = request.args.get('installmentPeriod')
-
-    campaign_id = request.args.get('campaignId', default='')
-
-    return render_template('blast-vip.html', form=form,
-            campaign_id=campaign_id, installment_period=installment_period,
-        openended_status='Open', amount=amount,
-        key=app.config['STRIPE_KEYS']['publishable_key'])
+    return redirect("https://checkout.texastribune.org/blastform", code=302)
```

Comment:
Might be worth making this a relative URL so that it redirects you to the local `blastform` when you're in dev mode.